src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2699,6 +2728,14 @@ public EntityResult tPerformer22Practitioner(Performer2 cdaPerformer2, IBundleIn
 		return tAssignedEntity2Practitioner(cdaPerformer2.getAssignedEntity(), bundleInfo);
 	}
 
+	public EntityResult tPerformer12Practitioner(Performer1 cdaPerformer1, IBundleInfo bundleInfo) {
```

Comment:
you don't actually call this, you call tAssignedEntity2Practitioner. Is it necessary?  If you did mean to use it, be aware that you could potentially be making empty practitioner objects in the bundle and should check for that.

Reply:
What might create an empty practitioner?

Reply:
^yes basically. I don't think you would actually create one, just saying the function can return one. It wouldn't hurt to add a check there.

Reply:
Would it not have an assignedEntity?